designsystem/src/commonMain/kotlin/org/hisp/dhis/mobile/ui/designsystem/component/InputYesNoField.kt

```diff
@@ -0,0 +1,100 @@
+package org.hisp.dhis.mobile.ui.designsystem.component
+
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.outlined.Cancel
+import androidx.compose.material3.Icon
+import androidx.compose.material3.RadioButton
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.testTag
+import org.hisp.dhis.mobile.ui.designsystem.resource.provideStringResource
+import java.util.Locale
+
+/**
+ * DHIS2 Input Yes/No Field. Wraps DHIS · [RadioButton].
+ * @param title controls the text to be shown for the title
+ * @param modifier allows a modifier to be passed externally
+ * @param orientation Controls how the radio buttons will be displayed, HORIZONTAL for rows or
+ * VERTICAL for columns.
+ * @param state Manages the InputShell state
+ * @param supportingText is a list of SupportingTextData that
+ * manages all the messages to be shown
+ * @param legendData manages the legendComponent
+ * @param isRequired controls whether the field is mandatory or not
+ * @param itemSelected controls which item is selected.
+ * @param onItemChange is a callback to notify which item has changed into the block.
+ */
+@Composable
+fun InputYesNoField(
+    title: String,
+    modifier: Modifier = Modifier,
+    orientation: Orientation = Orientation.HORIZONTAL,
```

Comment:
I think that in the yes/no field the orientation is always horizontal, so we don't need to expose it.